version.yaml

```diff
@@ -18,7 +18,7 @@ mount=mount-buildingblock-gitversion-gen-dir:
 
 image=image-buildingblock-gitversion:
   image: elbb/bb-gitversion
-  tags: ["0.1.0"]
+  tags: ["0.2.0"]
```

Comment:
0.3.0 ?

Reply:
Not part of this PR, but i can change it.